RpcWallet/RpcWallet.cs

```diff
@@ -127,7 +128,7 @@ private void ProcessInvoke(JObject result)
                     Script = result["script"].AsString().HexToBytes(),
                     Gas = Fixed8.Parse(result["gas_consumed"].AsString())
                 };
-                tx.Gas -= Fixed8.FromDecimal(10);
+                tx.Gas -= Settings.Default.MaxGasInvoke;
```

Comment:
Should be 10 gas (the free 10 gas for each transaction)

Reply:
In fact, I did not understand this part very well.

Reply:
What is the best way to limit the `Wallet.MakeTransaction(tx);` gas limit simulation?

Reply:
It created an InvocationTransaction for the `sendrawtransaction`.

Reply:
Maybe add an extra parameter on the https://github.com/neo-project/neo/blob/cc0f315854a9cf09f2efda2f83cf1199c9ece4a9/neo/Wallets/Wallet.cs#L223?

Reply:
Or just try to add simulate that to  tx.SystemFee = Settings.Default.MaxGasInvoke;

Reply:
I still did not understand the direction to implement this.

Reply:
I think I will leave this closed, maybe this can be solved direction on Neo RpcServer.cs class.